src/main/java/com/microsoft/azure/springcloudplayground/controller/AbstractPlaygroundController.java

```diff
@@ -50,40 +51,37 @@ public void invalidProjectRequest(HttpServletResponse response,
 
     /**
      * Render the home page with the specified template.
+     *
      * @param model the model data
      */
     protected void renderHome(Map<String, Object> model) {
         GeneratorMetadata metadata = this.metadataProvider.get();
+        BeanWrapperImpl wrapper = new BeanWrapperImpl(metadata);
 
         model.put("serviceUrl", generateAppUrl());
-        BeanWrapperImpl wrapper = new BeanWrapperImpl(metadata);
+
         for (PropertyDescriptor descriptor : wrapper.getPropertyDescriptors()) {
             if ("types".equals(descriptor.getName())) {
                 model.put("types", removeTypes(metadata.getTypes()));
             }
             else {
-                model.put(descriptor.getName(),
-                        wrapper.getPropertyValue(descriptor.getName()));
+                model.put(descriptor.getName(), wrapper.getPropertyValue(descriptor.getName()));
             }
         }
 
         // Google analytics support
-        model.put("trackingCode",
-                metadata.getConfiguration().getEnv().getGoogleAnalyticsTrackingCode());
-
-    }
-
-    public Function<String, String> getLinkTo() {
-        return this.linkTo;
+        model.put("trackingCode", metadata.getConfiguration().getEnv().getGoogleAnalyticsTrackingCode());
```

Comment:
let's remove this?

Reply:
OK, that will the next step for cleanup. For simply, this PR is avoiding touch the details of code implementation.